feature-libs/customer-ticketing/components/customer-ticketing-list/customer-ticketing-list-header/customer-ticketing-list-header.component.html

```diff
@@ -0,0 +1,15 @@
+<h2 class="cx-title d-none d-lg-block d-xl-block">
+    <!-- {{ 'title' | cxTranslate }} -->
+</h2>
+
+<ng-container>
+    <div class="cx-header-actions">
+        <!-- <cx-customer-ticketing-list-header-actions></cx-customer-ticketing-list-header-actions> -->
```

Comment:
Following @bgambocjaviniar comment about the decoupling: 
Don't we have a separate CMS Component for the Add Ticket button? 
Is this button going to be placed in the Header or in the Table component? @bgambocjaviniar Does the placement of this button affects (or not) the decoupling?

Reply:
The placement of the button affects the decoupling because think of it as layers when reading from the backend in terms of cms components in the slots

Looking at the submitted sample data, it goes as follows

```
SupportTicketHistoryComponent 

then

SupportTicketCreateComponent
```

so my assumption right now is `support ticket history component` is the table that contains a header (title), and the `support ticket create component` shows the add ticket button component.

Again, think of them as layers and the 1on1 relationship between the backend cms component with the mapped component int he frontend

Reply:
Usually, in all listing pages in spartacus, we have 1 cms component that maps to a 'listing component'. That listing component has the title, sorting / pagination, and table.

For this case, I believe we allow the flexibility to disallow people to add support tickets, which is why I believe it's a cms component right?

Reply:
----- 
as discussed in the call, we'll move forward with a cms history component mapping to a listing component, which contains all of the above (title, sorting, pagination, table, create new support ticket button)